awsf3/cloudwatch_agent_config.json

```diff
@@ -19,15 +19,15 @@
 				"measurement": [
 					"usage_active"
 				],
-				"metrics_collection_interval": 60,
+				"metrics_collection_interval": 120,
```

Comment:
I chose 120s instead of a larger interval here. For troubleshooting workflows (i.e., memory or storage issues) a 5 min interval might not collect enough data if there are sudden spikes.